### DISCUSSION TITLE
How to make edits sync across source and destination chats ?

### DISCUSSION THREAD

**Discussion by dennisdewaard** [aahnik/tgcf#179] [Q&A] [answered]
I'm forwarding a channel that has a bot running that edit a pinned message every minute.
Now I’m getting every minute a new message in the destination channel.
Is there a possibility to edit the message in the destination channel?
In the log I see a delete_sync and a delete_on_edit options.
Is that something I could use?

live=LiveSettings(delete_sync=False, delete_on_edit=None) past=PastSettings(delay=0) admins=[] plugins={}

**Comment by aahnik** [comment, marked as answer]
The original message must be written when the bot is running. Now when the original message is edited in source chat, the bot will edit it on the destination chat also.

Run the bot. Write a message in source chat. It will get forwarded to the destination chat. Now edit it. It will get edited in destination chat.

**Comment by dennisdewaard** [comment]
I'm not a admin of the source chat and they don't want to restart the bot.
Is there a way to create the original message?
Or can i run the chat in past mode an will it then get the original message

**Comment by aahnik** [comment]
> Is there a way to create the original message?

If a message is edited in source chat, and the corresponding message does not exist in the destination, a new message is sent automatically.

> Or can i run the chat in past mode an will it then get the original message

No. Past and live modes are completely separate. Using past mode you can forward the message. But on editing it, edit sync will not happen, as it was not forwarded by live mode.


I cant understand your scenario, or what you are trying to do. Please read the wiki pages for a better understanding of what tgcf does.
